setup.cfg

```diff
@@ -45,6 +45,8 @@ exclude_lines =
 omit =
     pytorch_lightning/cluster_environments/*.py
     pytorch_lightning/utilities/distributed.py
+    pytorch_lightning/trainer/evaluation_loop.py
+    pytorch_lightning/trainer/predict_loop.py
```

Comment:
there shall be some TODO, right?

Reply:
I have the pr to remove this ready together with old loop removal, will send right now.